assets/js/main.js

```diff
@@ -67,10 +67,13 @@ function makeTags(tagArray) {
 }
 function makeRow(video) {
   /* eslint-disable no-plusplus */
+  let authorRow = `<td class="author col-md-2"><a href="https://www.youtube.com/channel/${video.youtube_key}" name="${video.account_name}'s Channel" target="_blank" rel="noopener">`;
+  authorRow += video.author_ty_thumbnail ? `<img class='author-img' alt='${video.account_name}'>` : '';
```

Comment:
author_yt_thumbnail not ty

Reply:
Does this image tag have the src attribute for the thumbnail?

Reply:
ah shoot, no it doesn't. fixing!